setup.py

```diff
@@ -11,7 +11,11 @@
       version='0.10.0',
       description=u"Tools for testing rasters",
       long_description=long_description,
-      classifiers=[],
+      classifiers=[
+          'Topic :: Scientific/Engineering :: GIS',
+          'Programming Language :: Python :: 2.7',
+          'Programming Language :: Python :: 3.6'
+      ],
```

Comment:
Good idea, we should be adding the Python 3.6 classifier to all our projects.

Reply:
it's on our todo :-)